--- a/vendor/friendsofsymfony/jsrouting-bundle/Resources/webpack/FosRouting.js	2024-01-02 13:37:19.774099800 +0000
+++ b/vendor/friendsofsymfony/jsrouting-bundle/Resources/webpack/FosRouting.js	2024-01-02 13:57:28.444099800 +0000
@@ -5,7 +5,7 @@
 const path = require('path');
 const util = require('util');
 
-const InjectPlugin = require('@bpnetguy/webpack-inject-plugin').default;
+// const InjectPlugin = require('@bpnetguy/webpack-inject-plugin').default;
 
 const execFile = util.promisify(require('child_process').execFile);
 const readFile = util.promisify(fs.readFile);
@@ -90,11 +90,11 @@
             }
         });
 
-        new InjectPlugin(() => {
-            return 'import Routing from "fos-router";' +
-                'import routes from '+JSON.stringify(this.finalTarget)+';' +
-                'Routing.setRoutingData(routes);';
-        }).apply(compiler);
+        // new InjectPlugin(() => {
+        //     return 'import Routing from "fos-router";' +
+        //         'import routes from '+JSON.stringify(this.finalTarget)+';' +
+        //         'Routing.setRoutingData(routes);';
+        // }).apply(compiler);
     }
 }
 
